pkg/manager/member/executor_manager.go

```diff
@@ -73,9 +72,9 @@ func (m *executorMemberManager) Sync(ctx context.Context, tc *v1alpha1.TiflowClu
 		return nil
 	}
 
-	if !tc.MasterIsAvailable() {
-		return controller.RequeueErrorf("tiflow cluster: %s/%s, waiting for tiflow-master cluster running", ns, tcName)
-	}
+	// if !tc.MasterIsAvailable() {
```

Comment:
The logic here is currently invalid, because the Reconcile logic  must be before the Conditions Check